adoc/chapters/what_changed.adoc

```diff
@@ -373,8 +373,7 @@ The [code]#cl::sycl::byte# has been deprecated and now the {cpp17}
 [code]#std::byte# should be used instead.
 
 A SYCL implementation is no longer required to provide a host device.
-Instead, an implementation is only required to provide at least one
-device.  Implementations are still allowed to provide devices that are
+Implementations are still allowed to provide devices that are
```

Comment:
I'm actually not really happy with changing the changelog. Alternative ideas:

- Use ~~strikethrough~~ and add a sentence that this was later reverted.
- Add a new subsection `Changes made in revision 9` that details this.